### ISSUE TITLE
Games directory not working

### ISSUE THREAD

**Issue by SoWnoy** [lindstrm/Twitcher#27]
I can see my followed streamers, they show up, but when I try to check a game from the directory page, let's take StarCraft II for example, nothing shows up. It's just blank.

How can I solve this issue?

**Comment by lindstrm** [comment]
Sorry for super late answer, but Twitch probably had some problems with its api. 
Or are you still experiencing this issue?

**Comment by SoWnoy** [comment]
Yes I am still experiencing this issue. It happens every time I open the
program.

**Comment by lindstrm** [comment]
Please update to 2.1.15, i've included error reporting so then maybe i might be able to see what's going on since i can't reproduce.
